docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```
@@ -79,6 +82,36 @@ subquestion: |
 buttons:
   Log into this server: signin
 ---
+id: dangerous config var warning
```

```suggestion
id: dangerous config var warning
if: len(dangerous_config_var_names) > 0
```
This is optional, but I like to do it, considering it's an assumption you're making when displaying the page.